addons/stock/models/stock_move.py

```diff
@@ -780,7 +780,7 @@ def _prepare_procurement_values(self):
                 group_id = False
         return {
             'company_id': self.company_id,
-            'date_planned': self.date,
```

Comment:
By fixing the onchange, this should not be necessary anymore.  The onchange is the only thing also that seems different from what is in v9.